### PR TITLE
docs: API conclusion

### PR DESCRIPTION
**Pull request by benjlevesque**
## Link to Jira

https://requestnetwork.atlassian.net/browse/PROT-

### CODE REVIEW

File: packages/docs/docs/guides/3-Portal-API/5-api-conclusion.md [hunk 1]
```diff
@@ -9,10 +9,18 @@ description: >-
...
+* Faster response time
+* Notifications on new request and received payments
+* Send requests to other API users based on their email, instead of their ETH address.
+* Easy to build applications on top of Request (see [here](./3-api-apps))
+* Easy choice between fully public data or encrypted data on the network 
 
 ## Cons
-TODO
-* Your users have to create a Request account the first time they use the feature
+* centralised
```
Comment (rittme) [inline packages/docs/docs/guides/3-Portal-API/5-api-conclusion.md]: Nit: pros are all uppercase, cons are all lowercase :)

File: packages/docs/docs/guides/3-Portal-API/5-api-conclusion.md [hunk 2]
```diff
@@ -9,10 +9,18 @@ description: >-
...
 
 
-Need to explain how to connect their app: https://dashboard.request.network/settings/apps
+Request API is an abstraction over the Request Network for easy, but centralised, access. It's the fastest way to get started and build integrations.
+
 ## Pros
-TODO
+* Simple REST API, familiar for classic web developers
+* No key management
+* Faster response time
```
Comment (rittme) [inline packages/docs/docs/guides/3-Portal-API/5-api-conclusion.md]: With the buffer on the storage layer I'm not sure it's still the case. But I'm ok with leaving this here until we have measurements.

File: packages/docs/docs/guides/3-Portal-API/5-api-conclusion.md [hunk 3]
```diff
@@ -9,10 +9,18 @@ description: >-
 ---
 
 
-Need to explain how to connect their app: https://dashboard.request.network/settings/apps
+Request API is an abstraction over the Request Network for easy, but centralised, access. It's the fastest way to get started and build integrations.
```
Comment (ltacker) [inline packages/docs/docs/guides/3-Portal-API/5-api-conclusion.md]:
```suggestion
Request API is an abstraction over the Request Network for easy, but centralized, access. It's the fastest way to get started and integrate Request to your product.
```

File: packages/docs/docs/guides/3-Portal-API/5-api-conclusion.md (hunk 1 above)
Comment (ltacker) [inline packages/docs/docs/guides/3-Portal-API/5-api-conclusion.md]:
```suggestion
* centralized 
```

America f**k yeah

File: packages/docs/docs/guides/3-Portal-API/5-api-conclusion.md [hunk 4]
```diff
@@ -9,10 +9,18 @@ description: >-
...
+
 ## Pros
-TODO
+* Simple REST API, familiar for classic web developers
+* No key management
+* Faster response time
+* Notifications on new request and received payments
```
Comment (ltacker) [inline packages/docs/docs/guides/3-Portal-API/5-api-conclusion.md]:
```suggestion
* Notifications on new requests and received payments
```

File: packages/docs/docs/guides/3-Portal-API/5-api-conclusion.md [hunk 5]
```diff
@@ -9,10 +9,18 @@ description: >-
...
+* Send requests to other API users based on their email, instead of their ETH address.
+* Easy to build applications on top of Request (see [here](./3-api-apps))
+* Easy choice between fully public data or encrypted data on the network
```
Comment (ltacker) [inline packages/docs/docs/guides/3-Portal-API/5-api-conclusion.md]: What do you mean ?
Do you mean it's easy to create encrypted requests ?

File: packages/docs/docs/guides/3-Portal-API/5-api-conclusion.md (hunk 1 above)
Comment (ltacker) [inline packages/docs/docs/guides/3-Portal-API/5-api-conclusion.md]: Well, the payment is not centralized and the data is still stored on a distributed ledger.

I would say `not fully decentralized`

File: packages/docs/docs/guides/3-Portal-API/5-api-conclusion.md (hunk 2 above)
Comment (benjlevesque) [reply packages/docs/docs/guides/3-Portal-API/5-api-conclusion.md]: read performance is still better.

File: packages/docs/docs/guides/3-Portal-API/5-api-conclusion.md (hunk 5 above)
Comment (benjlevesque) [reply packages/docs/docs/guides/3-Portal-API/5-api-conclusion.md]: yes, but the request will not be end-to-end encrypted

File: packages/docs/docs/guides/3-Portal-API/5-api-conclusion.md (hunk 1 above)
Comment (benjlevesque) [reply packages/docs/docs/guides/3-Portal-API/5-api-conclusion.md]:
```suggestion
* Centralised 
```

File: packages/docs/docs/guides/3-Portal-API/5-api-conclusion.md [hunk 6]
```diff
@@ -9,10 +9,18 @@ description: >-
...
+* Notifications on new request and received payments
+* Send requests to other API users based on their email, instead of their ETH address.
+* Easy to build applications on top of Request (see [here](./3-api-apps))
+* Easy choice between fully public data or encrypted data on the network 
 
 ## Cons
-TODO
-* Your users have to create a Request account the first time they use the feature
+* centralised 
+* users need a Request account to connect
```
Comment (benjlevesque) [inline packages/docs/docs/guides/3-Portal-API/5-api-conclusion.md]:
```suggestion
* Users need a Request account to connect
```

File: packages/docs/docs/guides/3-Portal-API/5-api-conclusion.md [hunk 7]
```diff
@@ -9,10 +9,18 @@ description: >-
...
+* encryption is not end-to-end (see [here](./4-api-encryption.md))
```
Comment (benjlevesque) [inline packages/docs/docs/guides/3-Portal-API/5-api-conclusion.md]:
```suggestion
* Encryption is not end-to-end (see [here](./4-api-encryption.md))
```

**Pull request merged**